tmpl/rfc.ejs.html

```diff
@@ -104,13 +104,14 @@
         <div class="row">
           <div class="col-md-12">
             <div class="col-sm-4 col-xs-12">
-              <p>Interledger Team</p>
+              <p>Copyright © 2018<br/> W3C Interledger Payments CG</p>
```

Comment:
Is there any way to dynamically get this date? It's already outdated...unless you want it to be pegged to 2018?